tests/hierarchy_driver.cc

```diff
@@ -114,7 +130,7 @@ void matrix_based_two_grids(std::shared_ptr<boost::property_tree::ptree> params)
   Source<dim> source;
 
   auto laplace_ptree = params->get_child("laplace");
-  auto fe_degree = laplace_ptree.get<unsigned>("fe_degree", 1);
+  auto fe_degree = laplace_ptree.get<unsigned>("fe_degree", 4);
```

Comment:
What was the motivation for changing the default value here?

Reply:
I just wanted to be more close to what we use for profiling, but don't feel strongly about this.

Reply:
I changed it back.